languages/javascript/exercises/concept/booleans/.docs/hints.md

```diff
@@ -1,11 +1,14 @@
 ### 1. Check if the 'Fast Attack' action is possible
 
 - The logical NOT operator (`!`) can be placed before an expression to negate its value.
+- Check out [this MDN article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators#Logical_NOT_!) on Logical Operators
```

Comment:
Why is logical operators not capitalised on the point above, but is in this point?

Reply:
Because the article name is Logical Operators.